### PR TITLE
add decorators, add proptypes

### PR DESCRIPTION
**Pull request by sajdre**

### CODE REVIEW

File: src/components/ArticleList.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      <ul>
+        {articleElements}
+      </ul>
+    )
+}
+
+ArticleList.propTypes = {
+  articles: PropTypes.array.isRequired,
+  openedElementId: PropTypes.string,
+  toggleAccordion: PropTypes.func.isRequired
```
Comment (DzmitryU) [inline src/components/ArticleList.js]: Do you think that this element should know that we will put it inside accordion or any other decorators?

File: src/components/CommentsList.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Comment from './Comment'
+import toggleOpen from './../decorators/toggleOpen'
+import PropTypes from 'prop-types'
+
+function CommentsList({comments, isOpen, toggleOpen}) {
+  const commentsList = isOpen && comments ?
```
Comment (DzmitryU) [inline src/components/CommentsList.js]: What if we would add default value for comments list?

Comment (sajdre) [reply src/components/CommentsList.js]: added default props

File: src/components/ArticleList.js (hunk 1 above)
Comment (sajdre) [reply src/components/ArticleList.js]: changed name

**Pull request merged**